scripts/calc_bayesian_blocks.py

```diff
@@ -0,0 +1,36 @@
+from argparse import ArgumentParser
+
+from astropy.stats import bayesian_blocks
+from astropy.table import Table
+from gammapy.estimators import FluxPoints
+
+parser = ArgumentParser()
+parser.add_argument("-i", "--input-path", required=True)
+parser.add_argument("-o", "--output", required=True)
+parser.add_argument("--threshold", default=0.0027, type=float)
+args = parser.parse_args()
+
+
+def main(input_path, output, threshold):
+    lc = FluxPoints.read(input_path, format="lightcurve")
+    table = lc.to_table(format="lightcurve", sed_type="flux")
+
+    # No error on t
+    t_min = table["time_min"]
+    t_max = table["time_max"]
+    t = (t_min + t_max) / 2
```

Comment:
Why no error on t? Could it be center+-width/2 ? Is it necessary? Something for later?

Reply:
Can you use the error on t in the bayesian_block method / the astropy implementation? I have only found sigma as the error on x.